packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache

```diff
@@ -41,11 +44,14 @@
       {{/isFirefoxAndroid}}
       {{#isFirefoxDesktop}}
         <p id="install-mobile-firefox-desktop">
-          {{#isSignIn}}
-            {{#t}}Still adding devices?{{/t}}
-          {{/isSignIn}}
-          {{#t}}Sign in to Firefox on another device to complete set-up{{/t}}
+          {{#isSignedIn}}
+            <div class="button-row">
+              <a id="sync-firefox-devices" href="{{{pairingUrl}}}" class="button primary-button">{{#t}}Sync Firefox across devices{{/t}}</a>
+            </div>
+          {{/isSignedIn}}
         </p>
+        <div class="pair-everywhere-cta text-[#737373] text-xs font-normal leading-[18px]">{{#t}}Sign in everywhere you use Firefox.{{/t}}</div>
+        <div class="pair-everywhere-cta text-[#737373] text-xs font-normal leading-[18px]">{{#t}}We’re on Android and iOS, too!{{/t}}</div>
```

Comment:
Per https://tailwindcss.com/docs/line-height#fixed-line-heights
Looks like `leading-4` is 16px and `leading-5` is 20px. Not sure if either of those are <i>close enough<sup>(tm)</sup></i> to avoid needing a custom style.

Reply:
And I guess same comment as above re: custom `bg-[]` color. I think this is currently the only instance of us using a custom Tailwind `text-[]` color:
```sh
git grep -EIon "(bg|text)-\[.*?\]" packages | cat

packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache:6:bg-[#B3FFE3] rounded-[50%]
packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache:53:text-[#737373] text-xs font-normal leading-[18px]
packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache:54:text-[#737373] text-xs font-normal leading-[18px]
```